tango/server.py

```diff
@@ -1173,18 +1181,22 @@ class device_property(_BaseProperty):
         class PowerSupply(Device):
 
             host = device_property(dtype=str)
+            port = device_property(dtype=int, mandatory=True)
 
     :param dtype: Data type (see :ref:`pytango-data-types`)
     :param doc: property documentation (optional)
     :param default_value: default value for the property (optional)
+    :param mandatory (optional: default is False)
```

Comment:
The order of the arguments here doesn't match the order in the function prototype